src/components/SearchPage/SearchPage.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import HeaderContainer from "../HeaderContainer/HeaderContainer";
+import SearchResultsContainer from '../SearchResultsContainer/SearchResultsContainer';
+
+const SearchOptions = {
```

Comment:
how about array of options with key (as in the films data.json) and value (as displayed for the user)

Reply:
Just a question: why is it better?